tests/phpunit/api/v3/ContributionTest.php

```diff
@@ -2485,12 +2485,6 @@ public function testCompleteTransactionMembershipPriceSet() {
     $logs = $this->callAPISuccess('MembershipLog', 'get', array(
       'membership_id' => $this->_ids['membership'],
     ));
-    //CRM-19600: Ensure that 'Membership Renewal' activity is created after successful membership regsitration
```

Comment:
Earlier after the successfully payment of a pending membership, membership renewal was created which was incorrect. As per the fix https://github.com/civicrm/civicrm-core/pull/9468/files#diff-7a5b0e2d131dabc49178460fc63328c0L4603 the expected behaviour would be to change the 'Scheduled' Membership Signup Activity to 'Completed'. Reason why I have removed this test-case.